doc/glossary.rst

```diff
@@ -222,6 +225,15 @@ general neuroimaging concepts. If you think a term is missing, please consider
         anatomically (based on physical structure of the cortex) or functionally
         (based on cortical response to specific stimuli).
 
+    last_samp
+        The :attr:`~io.Raw.last_samp` attribute of :class:`~io.Raw`
+        objects is an integer representing the number of time samples that
+        passed between the onset of the hardware acquisition system and the
+        time when data stopped to be recorded to disk. This approach to sample
```

Comment:
```suggestion
        time when data started to be recorded to disk. This approach to sample
```

Reply:
This one is last_samp, not first_samp, that I added since both are mentioned across the docstrings.

Reply:
Oops. Sorry